components/pixels/lib.rs

```diff
@@ -64,10 +64,16 @@ pub fn rgba8_byte_swap_colors_inplace(pixels: &mut [u8]) {
 pub fn rgba8_byte_swap_and_premultiply_inplace(pixels: &mut [u8]) {
```

Comment:
We should replace this function with `generic_transform_inplace::<1, true, false>(pixels)` anyway.

Reply:
Replace it inside this PR?

Reply:
Hm, lets make it in this PR, this way we ensure that it's tested.

Reply:
Done